site/content/en/docs/api/extension_types.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "API Reference"
+title = "Envoy Gateway API"
```

Comment:
should this be "Gateway API Extensions" instead ?

Reply:
and we can add a description here saying - Envoy Gateway provides these extensions to support additional features not available in the Gateway API today

Reply:
+1 for this, done.